HRTool/src/main/java/app/controllers/ReportRestController.java

```diff
@@ -155,21 +161,27 @@ else if(tc.getTipTagCloud().equals(TagCloudEnum.IDE)) {
 		HttpHeaders headers = new HttpHeaders();

```

Comment:
You should generally not return file not byte[] as response type. If your method is generating a file, generate it on the file server and return the url through which the file can be accessed.